toolkit/tools/imagecustomizerapi/config.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+type Config struct {
+	SystemConfig SystemConfig `yaml:"SystemConfig"`
```

Comment:
Not sure if it makes sense here, but we support a list of configs (mostly as a feature of the ISO installer which can offer a selection of different configs to the user to install).

Reply:
For the image customizer, having a list didn't make much sense and might have caused user confusion. So, I removed it.

Reply:
Is it a goal or non-goal to keep the schemas in sync? Naively, I'd assume that compatibility of interface would be a good thing. If there's a desire to make this more of a singleton and less of a list... is there a way to adapt the existing logic in the core toolkit to accept either a single object *or* a list? That would permit greater compatibility of the format.

Reply:
Not perfectly in sync, no. But the intention is to avoid having differences without good cause.

I could probably extend the image creation schema to also support both `SystemConfig` and `SystemConfigs`.

Reply:
@reubeno, PR for adding `SystemConfig` property to the new image config as well: #6393